experiment/scheduler.py

```diff
@@ -188,34 +187,17 @@ class TrialInstanceManager:  # pylint: disable=too-many-instance-attributes
     handle_preempted_trials method should be called in the scheduling loop.
     See the docstring for handle_preempted_trials for how it works.
     """
-    # Hard limit on the number of nonpreemptibles we will use. This bounds
-    # costs.
-    MAX_NONPREEMPTIBLES = 500
-
-    # The maximum fraction of total trials in the experiment that can be done
-    # using nonpreemptibles. This helps bound the cost in unexpected situations.
-    NONPREEMPTIBLES_FRACTION = 1 / 20
-
-    MAX_FRACTION_FOR_NONPREEMPTIBLES = 1 / 4
-
     # How many times the number of trials we need preemptibles can we launch.
-    MAX_PREEMPTIBLES_MULTIPLIER = 2
+    MAX_PREEMPTIBLES_MULTIPLIER = 4
```

Comment:
Since pre n1-standard-1 is 0.01 vs $0.04749975, we can easily cap it to 4x. I dont think eviction rate is anywhere near this, so we will never reach near it.

Reply:
I frankly don't know if a multiplier is needed anymore. It was very specific to the previous algorithm where we had a guaranteed bounded cost in the worst case scenario.

Reply:
I think this seems overly cautious to keep this, i think we can remove it as if there are too many preemptions in the first day, it is fine as well, we can continue to keep recreating bots, no need to bound it (as costs will still be low even if we keep respinning vms).